USE_CASES.md

```
@@ -0,0 +1,297 @@
+One command. Any database. Any platform.
+
+```bash
+spindb create myapp --engine postgresql
+spindb create cache --engine redis
+spindb create vectors --engine qdrant
+```
+
+### Core Features
+
+| Feature | Description |
+|---------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)